### ISSUE TITLE
Bug Tapo H100 Smart Hub > 12 Devices

### ISSUE THREAD

**Issue by michaelkun** [petretiandrea/home-assistant-tapo-p100#650]
Last version of integracion

Device:

H100
por TP-Link
Firmware: 1.3.9 Build 230522 Rel.103415
Hardware: 1.0

HS Versions: 

Core 2023.12.1
Supervisor 2023.11.6
Operating System11.2

The problem began to happen the moment I exceeded 12 devices paired with the tapo hub, from that moment every time I reload the integration devices randomly appear and disappear, always missing some, especially motion sensors or door sensors

log:

[home-assistant_tapo_2023-12-13T18-53-04.108Z.log](https://github.com/petretiandrea/home-assistant-tapo-p100/files/13664547/home-assistant_tapo_2023-12-13T18-53-04.108Z.log)

**Currently the sensor of a window is the one that does not appear, I understand that it is the binary_sensor error**

My theory is that when all the devices that depend on the coordinator are listed, exceeding a certain amount will cause the hub to time out and stop responding as expected. If this is the case, perhaps making a slower acquisition of the subdevices may work.

**Comment by alexskynet** [comment]
Similar problem here: in my case the HUB stops responding on the lan but is accessible from the Tapo App

**Comment by michaelkun** [comment]
@alexskynet Of course, from the tapo app everything works and appears correctly, it seems to be a rejection towards the integration

**Comment by Saftlad** [comment]
Issue is still ongoing.  I have now reverted to purchasing additional hubs to work around this limitation but it would be good if it could be resolved
